config/sources/families/include/rockchip64_common.inc

```diff
@@ -285,6 +285,14 @@ family_tweaks()
 
 		cp -R $SRC/packages/blobs/rtl8723bt_fw/* $SDCARD/lib/firmware/rtl_bt/
 		cp -R $SRC/packages/blobs/station/firmware/* $SDCARD/lib/firmware/
+		if [[ $BRANCH == legacy && ($BOARD == station-m1) ]]; then
```

Comment:
Probably without ()

Reply:
I left them in case the next model is added (maybe it will be P1)